rllib/tests/agents/parameters.py

```diff
@@ -371,6 +371,46 @@ def astuple(self):
             threshold=150.0,
             frameworks=[Framework.TensorFlow],
         ),
+        TestAgentParams.for_moab_move_to_center(
```

Comment:
This test is not testing any of the new regularizers. What is the purpose of the test?

Reply:
The regularizer was disabled by mistake. Enabled the test with VIB regularizer.